### PR TITLE
app.js not completed yet

### PR DESCRIPTION
**Pull request by gamboagirl**
This is what I have so far. I created the package.json, and updated trucks.js as well. I am working on app.js, but working through errors. I'll update when I can get it working.

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -2,3 +2,58 @@
...
 //  can also serve up css files from the public directory this way if you want)
 //  you need to support a '/trucks' endpoint, and a dynamic route for '/trucks/:name'
+
+var express = require('express');
+var app = express();
+var foodTrucks = require('./trucks');
+
+app.use(express.static('public'));
+
+app.get('/', function (request, response) {
```
Comment (jamdon) [inline app.js]: This is unnecessary. Using express.static as middle-ware processes the root route so this code won't do anything.

File: app.js [hunk 2]
```diff
@@ -2,3 +2,58 @@
...
+    response.redirect('/foodTrucks');
+});
+
+app.get('/trucks/:name', function (request, response) {
+    var truck = foodTrucks.getTruck(request.params.name);
+    var list;
+    
+    if (truck.name) {
+        list += "<li>Name: " + truck.name;
```
Comment (jamdon) [inline app.js]: Browsers are forgiving with markup, but you should add the closing li tag when constructing your markup.

**Comment by jamdon** [comment]
It's looking pretty good so far.

**Comment by gamboagirl** [comment]
I updated the files and synced, but it doesn't seem like I can create a new pull request. Does it automatically update with my new commits?

**Comment by jamdon** [comment]
Yes, it automatically updates because the pull request is simply linking back to your repository as opposed to a snapshot of your repository when you initiated the pull request.

**Comment by jamdon** [comment]
:+1:

**Pull request closed without merging**